packages/bot-web-ui/src/components/dashboard/dashboard-component/load-bot-preview/index.scss

```diff
@@ -0,0 +1,365 @@
+
+                &--show {
+                    opacity: 1;
+                    width: fit-content;
+                }
+            }
+        }
+
+        &-item {
+            display: grid;
+            grid-template-columns: 1fr 1fr 1fr 1fr;
+            grid-template-areas: ('text location');
+            padding: 0.8rem;
+            align-items: center;
+            text-align: center;
+
+            &:not(.load-strategy__recent-item--selected):hover {
+                background-color: var(--state-hover);
+                cursor: pointer;
+            }
+
+            &:not(:last-child) {
+                border-bottom: solid 0.1rem var(--border-normal);
+            }
+
+            &--selected {
+                background-color: var(--state-active);
+            }
+
+            &-text {
+                flex-direction: column;
+                display: flex;
+                justify-content: center;
+            }
+
+            &-title {
+                font-size: var(--text-size-s);
+            }
+
+            &-time {
+                font-size: var(--text-size-xxs);
+            }
+
+            &-location {
+                flex-direction: row;
+                display: flex;
+                align-items: center;
+                justify-content: center;
+            }
+
+            &-saved {
+                margin-left: 1rem;
+                font-size: var(--text-size-s);
+                line-height: 1.43;
+            }
+
+            &__button {
+                display: flex;
+
+                &__edit {
+                    margin-right: 0.8rem;
+                }
+
+                &__save {
+                    margin-right: 0.8rem;
+                }
+            }
+        }
+    }
+
+    &__local {
+        &-dropzone {
+            @include desktop {
+                padding-top: 1.6rem;
+            }
+
+            @include mobile {
+                height: 100%;
+                padding: 1.6rem;
+            }
+
+            &-area {
+                align-items: center;
+                display: flex;
+                flex-direction: column;
+                justify-content: center;
+                border: dashed 0.2rem var(--border-normal);
+                border-radius: $BORDER_RADIUS;
+                height: 100%;
+                padding: 1.6rem;
+            }
+        }
+
+        &-icon {
+            margin-bottom: 1.6rem;
+        }
+
+        &-title {
+            margin-bottom: 1.6rem;
+            font-size: var(--text-size-s);
+        }
+
+        &-description {
+            margin-bottom: 1.6rem;
+            font-size: var(--text-size-xs);
+        }
+
+        &-preview {
+            display: flex;
+            flex-direction: column;
+            position: relative;
+
+            @include mobile {
+                padding: 1.6rem;
+                height: calc(100% - 7.4rem); // - footer height
+            }
+
+            &-close {
+                // TODO : Dark Theme
```

Comment:
dark theme has been merged we can have it

Reply:
done ✅